selfdrive/modeld/models/nav.cc

```diff
@@ -0,0 +1,73 @@
+#include "selfdrive/modeld/models/nav.h"
+
+#include <cstdio>
+#include <cstring>
+
+#include "common/mat.h"
+#include "common/modeldata.h"
+#include "common/params.h"
+#include "common/timing.h"
+#include "system/hardware/hw.h"
+
+
+template<class T, size_t size>
+constexpr const kj::ArrayPtr<const T> to_kj_array_ptr(const std::array<T, size> &arr) {
```

Comment:
looks like this is copy and pasted from models/driving.cc?

Reply:
It is, not sure where a good place to put it would be?